Library/Homebrew/cask/cask.rb

```diff
@@ -122,6 +122,22 @@ def outdated_versions(greedy = false)
       installed.reject { |v| v == version }
     end
 
+    def outdated_info(greedy, verbose, json)
+      return token unless verbose || json
```

Comment:
```suggestion
      return token if !verbose && !json
```

is a bit easier to follow